scripts/opof_setup

```diff
@@ -8,7 +8,7 @@ COLOR_GREEN="\033[0;32m"
 COLOR_OFF="\033[0m"
 
 OVS_BRIDGE0=${OVS_BRIDGE1:-"ovsbr0"}
-OVS_BRIDGE0_PORTS=${OVS_BRIDGE0_PORTS:-"p0 pf0vf0 pf0sf0 pf0vf1"}
+OVS_BRIDGE0_PORTS=${OVS_BRIDGE0_PORTS:-"p0 pf0vf0 en3f1pf1sf0 pf0vf1"}
```

Comment:
en3f1pf1sf0 is for the sf on 2nd pf. For 1st pf should be en3f0pf0sf0.
Are you trying to make it work on lasted BFB?